whois-endtoend/src/test/java/net/ripe/db/whois/query/InverseQueryTestIntegration.java

```diff
@@ -266,7 +275,79 @@ public void inverse_email_from_untrusted_then_fail() {
         assertThat(response, containsString("is not an inverse searchable attribute"));
     }
 
+    @Test
+    public void inverse_email_UTF8_charset() {
```

Comment:
Use `utf8` in the method name for consistency

Reply:
I changed the name to this method
inverse_email_charset